packages/conventional-commits-parser/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
its a good idea to define base tsconfig on root level of project and use extends here